src/features/views/components/ViewDataTable/columnTypes/index.ts

```diff
@@ -45,7 +45,7 @@ const columnTypes: Record<COLUMN_TYPE, IColumnType> = {
   [COLUMN_TYPE.JOURNEY_ASSIGNEE]: new DummyColumnType(),
   [COLUMN_TYPE.LOCAL_BOOL]: new SimpleColumnType(),
   [COLUMN_TYPE.LOCAL_PERSON]: new LocalPersonColumnType(),
-  [COLUMN_TYPE.LOCAL_SMART_SEARCH]: new LocalSmartSearchColumnType(),
+  [COLUMN_TYPE.LOCAL_QUERY]: new LocalQueryColumnType(),
```

Comment:
Will you also replace the dummy type class for the `PERSON_QUERY` column, using the same as here?

Reply:
Aaaaaah i forgot!